addon/components/select-component.js

```diff
@@ -345,8 +349,7 @@ export default Ember.Component.extend(
       result.pushObjects(groupObj.members);
     });
     return result;
-  }).property('preparedContent.[]', 'optionGroupPath', 'labels.[]',
-    'groupSortFunction'),
+  }).property('preparedContent', 'preparedContent.[]', 'optionGroupPath', 'labels.[]', 'groupSortFunction'),
```

Comment:
You don't need `'preparedContent'` and `'preparedContent.[]'` The latter should be enough